controller/cmd/identity/main.go

```diff
@@ -39,7 +41,7 @@ func Main(args []string) {
 	controllerNS := cmd.String("controller-namespace", "", "namespace in which Linkerd is installed")
 	identityScheme := cmd.String("identity-scheme", "", "scheme used for the identity issuer secret format")
 	trustDomain := cmd.String("identity-trust-domain", "", "configures the name suffix used for identities")
-	encodedIdentityTrustAnchorPEM := cmd.String("identity-trust-anchors-pem", "", "Base64 encoded trust anchors certificate")
+	encodedIdentityTrustAnchorPEM := cmd.String("identity-trust-anchors-pem", os.Getenv(envTrustAnchors), fmt.Sprintf("Base64 encoded trust anchors certificate (%s)", envTrustAnchors))
```

Comment:
To keep things simple, since we're now reading this value from an env variable, we should just get rid of the flag for it entirely.  Easier to only read this value from one place.

Reply:
Done. Actually it would be nicer if the trust anchors would not need to be base64-encoded but some other tests fails when I change that. Therefore (and since I have to transform the config and trust anchors Secret within the cluster dynamically anyway within my kustomization) I leave it base64 encoded.